app/components/orders/order-summary.js

```diff
@@ -14,6 +14,10 @@ export default Component.extend(FormMixin, {
     );
   }),
 
+  orderStatus: computed('data.status', function() {
```

Comment:
just use this.data.status wherever this is required, this is redundant.

Reply:
Updating it now